lib/workflow/client.js

```diff
@@ -133,19 +133,25 @@ class Workflow {
   }
 
   profileTasks(profileId, establishmentId) {
-    const params = {};
-    if (establishmentId) {
-      params.query = { establishmentId };
-    }
-    return this.client(`/profile-tasks/${profileId}`, params);
+    const query = {
+      model: 'profile-touched',
+      modelId: profileId,
+      onlyOpen: true,
```

Comment:
I believe it was defaulting to open tasks before, because I couldn't find anywhere `all` was passed:
https://github.com/UKHomeOffice/asl-workflow/blob/master/lib/router/profile-tasks.js#L26-L29

Reply:
Ah ok, found it:
https://github.com/UKHomeOffice/asl-internal-api/blob/master/lib/routers/profile.js#L108

Reply:
I was about to link that same line of code.

https://github.com/UKHomeOffice/asl-internal-api/blob/master/lib/routers/profile.js#L108

Reply:
The only request using 'all' has been replaced with the related-tasks router, so we should be ok to carry on defaulting to onlyOpen for the profile tasks function.